osf/migrations/0170_add_ab_testing_home_page_version_b_flag.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-10-08 13:01
+from __future__ import unicode_literals
+from django.db import migrations
+
+from osf import features
+from osf.utils.migrations import AddWaffleFlags
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0169_merge_20190618_1429'),
+    ]
+
+    operations = [
+        AddWaffleFlags([features.EMBER_AB_TESTING_HOME_PAGE_VERSION_B]),
+    ]
```

Comment:
Sandshrew has been released since this PR was created, so you'll have to shift this migration to the end (preferred over adding a merge migration).

Reply:
☝️

Reply:
Be sure to remove this outdated migration @baylee-d !